trunk/SUAVE/Methods/Geometry/Two_Dimensional/Cross_Section/Airfoil/import_airfoil_geometry.py

```diff
@@ -19,7 +20,7 @@
 import scipy.interpolate as interp
 
 ## @ingroup Methods-Geometry-Two_Dimensional-Cross_Section-Airfoil
-def  import_airfoil_geometry(airfoil_geometry_files, npoints = 100):
+def  import_airfoil_geometry(airfoil_geometry_files, npoints = 200,surface_interpolation = 'linear'):
```

Comment:
I would do cubic as the default and provide in the docstring a link to the scipy documentation what the options are

Reply:
Link added and default changed. Note that this will change a lot of regressions